.circleci/config.yml

```diff
@@ -75,6 +75,14 @@ jobs:
           name: Unit Test
           command: |
             . venv/bin/activate
+            NODE_VERSION=16.13.0
+            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+            export NVM_DIR=$HOME/.nvm
+            . "$NVM_DIR/nvm.sh"
+            nvm install ${NODE_VERSION}
+            nvm use v${NODE_VERSION}
+            nvm alias default v${NODE_VERSION}
+            export PATH="$NVM_DIR/versions/node/v${NODE_VERSION}/bin/:${PATH}"
```

Comment:
this is using using the sceptre-circleci docker image and that [image is based on alpine](https://github.com/Sceptre/sceptre-circleci/blob/master/Dockerfile#L1) therefore you probably want to follow [directions to install on alpine distro](https://superuser.com/questions/1125969/how-to-install-npm-in-alpine-linux).